src/edu/ucsb/nceas/metacat/dataone/v1/CNodeService.java

```diff
@@ -93,7 +93,7 @@
 	private edu.ucsb.nceas.metacat.dataone.CNodeService impl = null;
 
 	/* the logger instance */
-	private Logger logMetacat = null;
+	private org.apache.commons.logging.Log logMetacat = null;
```

Comment:
```suggestion
	private org.apache.commons.logging.Log logMetacat = LogFactory.getLog(CNodeService.class);
```

Reply:
See the commit 70db73cab0ce506277782cd20f081b0dabcd9c8a.